common/physics-system.cpp

```diff
@@ -335,15 +335,45 @@ namespace tec {
 		for (int i = 0; i < data->entity.components_size(); ++i) {
 			const proto::Component& comp = data->entity.components(i);
 			switch (comp.component_case()) {
-			case proto::Component::kCollisionBody:
-			{
-				CollisionBody* collision_body = new CollisionBody();
-				collision_body->In(comp);
-				CollisionBodyMap::Set(entity_id, collision_body);
-				collision_body->entity_id = entity_id;
-				AddRigidBody(collision_body);
-			}
-			break;
+				case proto::Component::kCollisionBody:
+				{
+					CollisionBody* collision_body = new CollisionBody();
+					collision_body->In(comp);
+					CollisionBodyMap::Set(entity_id, collision_body);
+					collision_body->entity_id = entity_id;
+					AddRigidBody(collision_body);
+				}
+                    break;
+                case proto::Component::kRenderable:
+                    break;
```

Comment:
Most of these breaks can be removed as the cases will falldown till exit regardless.